docs/MUTATION_TESTING.md

```
@@ -0,0 +1,399 @@
+- `lib/features/order/models/`
+- `lib/features/order/providers/`
+
+**Auth:**
+- `lib/features/auth/notifiers/`
+- `lib/features/auth/providers/`
+
+#### 4. Increase threshold gradually
+
+Cada sprint, aumenta el threshold en 5%:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)